src/modm/driver/adc/ads868x.lb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Niklas Hauser
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+
+def init(module):
+    module.name = ":driver:ads868x"
+    module.description = """\
+# ADS868x ADC
+
+"""
```

Comment:
Short summary with link to datasheet?
You also want to add that this is a blocking driver, not an async one.

Reply:
Ok, done.